core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMergeSink.java

```diff
@@ -51,15 +51,16 @@ public interface ConnectorMergeSink
     /**
      * Store the page resulting from a merge. The page consists of {@code n} channels, numbered {@code 0..n-1}:
      * <ul>
-     *     <li>Blocks {@code 0..n-3} in page are the data columns</li>
-     *     <li>Block {@code n-2} is the tinyint <i>operation</i>:
+     *     <li>Blocks {@code 0..n-4} in page are the data columns</li>
+     *     <li>Block {@code n-3} is the tinyint <i>operation</i>:
      *     <ul>
      *         <li>{@link #INSERT_OPERATION_NUMBER}</li>
      *         <li>{@link #DELETE_OPERATION_NUMBER}</li>
      *         <li>{@link #UPDATE_OPERATION_NUMBER}</li>
      *         <li>{@link #UPDATE_INSERT_OPERATION_NUMBER}</li>
      *         <li>{@link #UPDATE_DELETE_OPERATION_NUMBER}</li>
      *     </ul>
+     *     <li>Block {@code n-2} is the merge case number, the number usually for connectors perform partial update</li>
```

Comment:
What's the type of this block? Also, I don't understand the second part of this sentence:
> the number usually for connectors perform partial update